.github/CONTRIBUTING.md

```diff
@@ -148,3 +148,18 @@ Possible log levels are:`trace`, `debug`, `info`, `warn`, `err`, `critical`, `of
   * The option needs to be added in `examples/md-flexible/src/parsing/MDFlexConfig.h`.
   * Parsing for it in `examples/md-flexible/src/parsing/CLIParser.cpp` and `YamlParser.cpp`
   * Make sure that the description is parsable by `CLIParser::createZSHCompletionFile()`
+
+### Making an Option tunable
+* If not already done, add a new setter to `src/autopas/AutoPas.h`.
+* Initiate the set of allowed options to all options in the constructor of AutoPas.
+* Add your option to `src/autopas/selectors/Configuration.h` and adjust constructors, comparison operators and ConfigHash function accordingly.
+* Add a parameter for your option to `TuningStrategyFactory::generateTuningStrategy` and pass it to the constructors for each tuning strategy.
+* Adjust the individual tuning strategies accordingly; the exact implementation will depend on the purpose of your option, but some general advice is:
+  * `FullSearch` and `PredictiveTuning` inherit from `SetSearchSpaceBasedTuningStrategy`, adjust the constructor for this class and the method `populateSearchSpace()`.
+  * For `ActiveHarmony` your option will also have to be integrated into `FeatureVector`.
```

Comment:
Wait no. `FeatureVector` is used for Bayesian-based strategies. Active Harmony must be told about the new parameter via `configureTuningParameter()`.